CONTRIBUTING.md

```diff
@@ -51,6 +51,79 @@ For source code integrity , Hyperledger Bevel GitHub pull requests are accepted
 8. Work on the feedbacks to revise the PR if there are any comments
 9. If not, the PR gets approved , delete feature branch post the merge
 
+## Create local branch
+
+> Whenever you begin work on a new feature or bugfix, it's important that you create a new branch.
+1. Clone your fork to your local machine
+
+2. Setup your local fork to keep up-to-date (optional)
+   ```
+   # Add 'upstream' repo to list of remotes
+   git remote add upstream https://github.com/hyperledger/bevel.git
+   # Verify the new remote named 'upstream'
+   git remote -v
+   # Checkout your main branch and rebase to upstream.
+   # Run those commands whenever you want to synchronize with main branch
+   git fetch upstream
+   git checkout main
+   git rebase upstream/main
+   ```
+3. Create your branch.
+   ```
+   # Checkout the main branch - you want your new branch to come from main
+   git checkout main
+   # Create a new branch named `<newfeature>` (give simple informative name)
+   git branch <newfeature>
+   ```
+4. Checkout your branch and add/modify files.
+   ```
+   git checkout <newfeature>
+   git rebase main
+   # Happy coding !
+   ```
+5. Commit changes to your branch.
+   ```
+   # Commit and push your changes to your fork
+   git add -A
+   git commit -s -m "<type>[optional scope]: <description>"
```

Comment:
I would rather change it to something like

`git commit -s`
(This opens the default text editor window, where you can write the commit title and message, along with a `fixes #issue_number` line at the end)

Reply:
See next review comment and you can added the commit title + body reference from there